lib/driverProviders/driverProvider.ts

```diff
@@ -89,42 +88,38 @@ export abstract class DriverProvider {
 
 
   /**
-   * Quits an array of drivers and returns a q promise instead of a webdriver one
+   * Quits an array of drivers and returns a native promise instead of a webdriver one
    *
    * @param drivers {webdriver.WebDriver[]} The webdriver instances
    */
-  static quitDrivers(provider: DriverProvider, drivers: WebDriver[]): q.Promise<void> {
-    let deferred = q.defer<void>();
-    wdpromise
-        .all(drivers.map((driver: WebDriver) => {
-          return provider.quitDriver(driver);
-        }))
-        .then(
-            () => {
-              deferred.resolve();
-            },
-            () => {
-              deferred.resolve();
-            });
-    return deferred.promise;
+  static quitDrivers(provider: DriverProvider, drivers: WebDriver[]): Promise<void> {
+    let deferredResolve: (x?: any) => void;
```

Comment:
Similar to below, we should just return a promise that wraps this code instead of using a deferred